src/dnx.windows/dnx.windows.vcxproj

```diff
@@ -30,6 +30,9 @@
   <PropertyGroup>
     <IncludePath Condition="'$(RuntimeTargetOS)' == 'WIN7_PLUS_CORESYSTEM'">$(ProgramFiles)\Windows Kits\Win8CoreSystem\inc\crt;$(IncludePath)</IncludePath>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
```

Comment:
This section isn't even needed, it's just because I toggled something that defaulted to true